tests/test_bot_unlike.py

```diff
@@ -0,0 +1,155 @@
+
+import pytest
+import responses
+
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
+
+from instabot.api.config import API_URL
+
+from .test_bot import TestBot
+from .test_variables import (TEST_CAPTION_ITEM, TEST_COMMENT_ITEM, TEST_PHOTO_ITEM,
+                             TEST_USERNAME_INFO_ITEM)
+
+
+class TestBotFilter(TestBot):
+
+    @responses.activate
+    @pytest.mark.parametrize('media_id,total,max_per_day', [
+        [111111, 1, 2],
+        [111111, 2, 2],
+        [111111, 3, 2],
+        ['111111', 1, 2],
+        ['111111', 2, 2],
+        ['111111', 3, 2]])
+    @patch('time.sleep', return_value=None)
+    def test_unlike(self, patched_time_sleep, media_id, total, max_per_day):
+        self.bot.total['unlikes'] = total
+        self.bot.max_per_day['unlikes'] = max_per_day
+        responses.add(
+            responses.POST, "{api_url}media/{media_id}/unlike/".format(
+                api_url=API_URL, media_id=media_id
+            ), json="{'status': 'ok'}", status=200
```

Comment:
What if we get "feedback_required"? Apparently "unlike" not handle this situation.